app/controllers/green_lanes/results_controller.rb

```diff
@@ -52,5 +56,14 @@ def c1ex
     def c2ex
       results_params[:c2ex]
     end
+
+    def permit_dynamic_ans(params)
+      if params.is_a?(Hash)
+        params.each_key do |key|
+          params[key] = params[key].is_a?(Hash) ? permit_dynamic_ans(params[key]) : params[key]
```

Comment:
is this recursive?
this looks complex, we should have at least a test for it.

Reply:
So I think this is effectively `permit!`